src/crackmapexec.yml

```diff
@@ -1,16 +1,23 @@
 ---
-# The CrackMapExec dependency gssapi requires the krb5-config
-# executable from the libkrb5-dev package.
+# The following CrackMapExec dependencies require the specified
+# packages be installed:
+# * gssapi requires the krb5-config executable from the libkrb5-dev
+#   package
+# * aardwolf requires a Rust compiler, as the Python package is only
+#   available from PyPI.org as a source distribution (i.e., there is
+#   no wheel available for download)
 - name: Install CrackMapExec dependencies
   ansible.builtin.package:
     name:
       - libkrb5-dev
+      - rustc
```

Comment:
Should we just install `cargo` since that's Rust's preferred method for environment set up?

Reply:
I'm really only interested in adding enough stuff for Python to build the wheel for `aardwolf`.  I could add `cargo`, but it doesn't seem to be necessary to build the wheel.

If the assessors requested us to add a Rust tool to this AMI, then I would modify [cisagov/ansible-role-assessment-tool](https://github.com/cisagov/ansible-role-assessment-tool) to support building a Rust project.  That would presumably involve installing `cargo`, and I would reuse that functionality here.  Do you want me to add an issue to that repo to support Rust (and reference this PR)?  Would scratch your `cargo` itch?

Reply:
That will work for me. I ran intro troubles where I needed to install `cargo` for a Python package to successfully build its Rust extensions so I just wanted to mention it as an option.

Reply:
@mcdonnnj, please see:
- cisagov/ansible-role-assessment-tool#41
- cisagov/ansible-role-assessment-tool#42
- fb16595
- 464f5d8